components/MobileNavigation.js

```diff
@@ -0,0 +1,167 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup'
+
+import LevitatingLink from './LevitatingLink'
+import Hamburger from '../resources/hamburger.svg'
+
+export default class MobileNavigation extends Component {
+  static defaultProps = {
+    navs: []
+  }
+
+  static propTypes = {
+    navs: PropTypes.arrayOf(PropTypes.shape({
+      href: PropTypes.string.isRequire,
+      labal: PropTypes.string.isRequire,
+      disabled: PropTypes.bool
+    }))
+  }
+
+  state = {
+    showMobileNav: false,
+    expandMobileNav: false
+  }
+
+  componentDidMount () {
+    window.addEventListener('scroll', this.handleScroll)
+  }
+
+  componentWillUnmount () {
+    window.removeEventListener('scroll', this.handleScroll)
+  }
+
+  handleScroll = (e) => {
+    if (window.scrollY > 700) {
+      if (!this.state.showMobileNav) this.showMobileNav()
+    } else {
+      if (this.state.showMobileNav) this.hideMobileNav()
+    }
+  }
+
+  showMobileNav = () => {
+    this.setState({
+      showMobileNav: true
+    })
+  }
+
+  hideMobileNav = () => {
+    this.setState({
+      showMobileNav: false,
+      expandMobileNav: false
+    })
+  }
+
+  handleToggleMobileNav = (value) => {
+    this.setState({
+      expandMobileNav: value || !this.state.expandMobileNav
+    })
+  }
+
+  handleClickNav = (e) => {
+    this.handleToggleMobileNav(false)
+  }
+
+  renderNav = () => {
+    const { navs } = this.props
+    const { expandMobileNav } = this.state
+
+    return (
+      <nav onClick={this.handleClickNav}>
```

Comment:
This navigation is for mobile, not for screen readers, so probably safe to add `aria-hidden` attribute here.